CHANGELOG.md

```diff
@@ -1,5 +1,232 @@
+- Send raw locations from the Publisher [\#250](https://github.com/ably/ably-asset-tracking-swift/issues/250)
+- Route is not recalculated when user is off route [\#184](https://github.com/ably/ably-asset-tracking-swift/issues/184)
+- Implement `RSC7d` \(Ably-Agent header\) [\#172](https://github.com/ably/ably-asset-tracking-swift/issues/172)
+- Implement stopping strategy [\#165](https://github.com/ably/ably-asset-tracking-swift/issues/165)
+- Test issue ignore [\#163](https://github.com/ably/ably-asset-tracking-swift/issues/163)
+- Send skipped location updates [\#135](https://github.com/ably/ably-asset-tracking-swift/issues/135)
+- Add token based authentication [\#134](https://github.com/ably/ably-asset-tracking-swift/issues/134)
+- Modify Publisher example app to work in background [\#110](https://github.com/ably/ably-asset-tracking-swift/issues/110)
+- Enable setting a constant location engine resolution [\#277](https://github.com/ably/ably-asset-tracking-swift/pull/277) ([lukasz-szyszkowski](https://github.com/lukasz-szyszkowski))
+
+**Fixed bugs:**
+
+- Subscriber: no asset connection status changes [\#314](https://github.com/ably/ably-asset-tracking-swift/issues/314)
+- Publisher.track method doesn't work as described in the code docs [\#308](https://github.com/ably/ably-asset-tracking-swift/issues/308)
+- Remove `fastlane` [\#202](https://github.com/ably/ably-asset-tracking-swift/issues/202)
+- DefaultAblyPublisherService never calls `.success` on channel publish [\#190](https://github.com/ably/ably-asset-tracking-swift/issues/190)
+- Add AAT library versioning [\#189](https://github.com/ably/ably-asset-tracking-swift/issues/189)
+- Specify channel modes to limit the amount of unnecessary data being sent and received over Ably [\#179](https://github.com/ably/ably-asset-tracking-swift/issues/179)
+- Release to Swift Package Manager \(SPM\) [\#148](https://github.com/ably/ably-asset-tracking-swift/issues/148)
+- Change names of client type values from presence data [\#136](https://github.com/ably/ably-asset-tracking-swift/issues/136)
+- EnhancedLocationUpdateMessage should be sent and received as a single message, not an array. [\#126](https://github.com/ably/ably-asset-tracking-swift/issues/126)
+- Use Ably and channel state when determining the trackable state in the Subscriber and Publisher SDK [\#120](https://github.com/ably/ably-asset-tracking-swift/issues/120)
+- The trackable state should be updated in the Publisher only after the state is different from the previous one. [\#115](https://github.com/ably/ably-asset-tracking-swift/issues/115)
+- Allow to call Publisher track\(\) method multiple times [\#112](https://github.com/ably/ably-asset-tracking-swift/issues/112)
+- Handle stopped state of the Subscriber [\#109](https://github.com/ably/ably-asset-tracking-swift/issues/109)
+- Fix lint errors for podspec [\#41](https://github.com/ably/ably-asset-tracking-swift/issues/41)
```

Comment:
This has also been just closed without fixing, so let's remove it from the list :wink: